### DISCUSSION TITLE
Change to webhooks

### DISCUSSION THREAD

**Discussion by dshannon-ifs** [osiset/laravel-shopify#1259] [General]
Hey all,

Can anyone advise if a merchant needs to reinstall the app for changes to webhook definitions to be picked up? I have removed some unnecessary webhooks that I was listening to. Is there any way for changes to webhook definitions to be picked up without reinstalling the app?

Thanks!

**Comment by Kyon147** [comment]
I think they will get picked up the next time the store goes through the `authenticate` route from memory.

**Reply by d-shannon** [reply]
That's great news. Thank you!
